endpoints.json

```diff
@@ -175,5 +175,24 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles?topic=cats": {
+    "description": "200: serves an array of articles, filtered by topic",
+    "queries": ["cats"],
+    "exampleResponse": {
+      "articles": [
+        {
+          "author": "butter_bridge",
+          "title": "UNCOVERED: catspiracy to bring down democracy",
+          "article_id": 5,
+          "topic": "cats",
+          "created_at": "2020-08-03T15:14:00.000Z",
+          "votes": 0,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          "comment_count": 2
+        }
+      ]
+    }
```

Comment:
This shouldnt be a completely new entry and should build from the original query, with the added:

`"queries": [..., "topic"]`
This is because the query name is topic, which should also include any other queries this endpoint takes (represented by the ...)